sqs/Module.md

```diff
@@ -92,300 +34,20 @@ sqs:Configuration configuration = {
     attributes["FifoQueue"] = "true";
 
-    string|error response1 = sqsClient->createQueue("demo.fifo", attributes);
-    if (response1 is string) {
-        log:printInfo("Created queue URL: " + response1);
-        // Keep the queue URL for future operations
-        queueResourcePath = sqs:splitString(response1, "amazonaws.com", 1);
-    }
-
-    // Send a message to the created queue
-    attributes = {};
-    attributes["MessageAttribute.1.Name"] = "Name1";
-    attributes["MessageAttribute.1.Value.StringValue"] = "Value1";
-    attributes["MessageAttribute.1.Value.DataType"] = "String";
-    attributes["MessageAttribute.2.Name"] = "Name2";
-    attributes["MessageAttribute.2.Value.StringValue"] = "Value2";
-    attributes["MessageAttribute.2.Value.DataType"] = "String";
-    string queueUrl = "";
-    sqs:OutboundMessage|error response2 = sqsClient->sendMessage("Sample text message.", queueResourcePath,
-        attributes, "grpID1", "dupID1");
-    if (response2 is sqs:OutboundMessage) {
-        log:printInfo("Sent message to SQS. MessageID: " + response2.messageId);
-    }
-
-    // Receive a message from the queue
-    string[] attributeNames = ["SenderId"];
-    string[] messageAttributeNames = ["N1", "N2"];
-    sqs:InboundMessage[]|error response3 = sqs->receiveMessage(fifoQueueResourcePath, 1, 600, 2, attributeNames, messageAttributeNames);
-    if (response3 is sqs:InboundMessage[] && response3.length() > 0) {
-        log:printInfo("Successfully received the message. Message body: " + response3[0].body);
-        log:printInfo("\nReceipt Handle: " + response3[0].receiptHandle);
-        // Keep receipt handle for deleting the message from the queue
-        receivedReceiptHandler = response3[0].receiptHandle;
+    string|error response = sqsClient->createQueue("demo.fifo", attributes);
+    if (response is string) {
+        log:printInfo("Created queue URL: " + response);
     }
-
-    // Delete the received the message from the queue
-    error? response4 = sqsClient->deleteMessage(queueResourcePath, receivedReceiptHandler);
-    if (response4 is ()) {
-        log:printInfo("Successfully deleted the message from the queue.");
-    }
-
-    // Delete the queue
-    error? response5 = sqsClient->deleteQueue(queueResourcePath);
-    if (response is ()) {
-        log:printInfo("Successfully deleted the queue.");
-    }
-}
-```
-### Pull the Module
-You can pull the Amazon SQS client from Ballerina Central:
-```shell
-$ bal pull ballerinax/aws.sqs
-```
-
-### Building the Source
-Execute the commands below to build from the source after installing Ballerina SLAlpha5 version.
-
-1. Clone this repository using the following command:
-    ```shell
-    $ git clone https://github.com/ballerina-platform/module-ballerinax-aws.sqs.git
-    ```
-
-2. Run this command from the `module-ballerinax-aws.sqs` root directory:
-    ```shell
-    $ bal build
     ```
+2. Use `bal run` command to compile and run the Ballerina program. 
```

Comment:
Shouldn't this be step 4?

Reply:
No it is under step 3, as other connector documentation